filter_issues.py

```
@@ -0,0 +1,8 @@
+import json
+
+with open('linear_issues.json', 'r') as f:
```

Ruff (0.15.5)

[warning] Unnecessary mode argument

Remove mode argument

(UP015)